src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.scss

```diff
@@ -264,6 +264,10 @@
 
   .cd-datatable-checkbox {
     text-align: center;
+    
+    &:checked {
+      accent-color: vv.$primary;
+    }
```

Comment:
The prettier linter might have an issue with this section based on:

```
.....................................................................................................................................................................................................................................................................................................A worker process has failed to exit gracefully and has been force exited. This is likely caused by tests leaking due to improper teardown. Try running with --detectOpenHandles to find leaks.


[lint:tslint  ] TSLint's support is discontinued and we're deprecating its support in Angular CLI.
[lint:tslint  ] To opt-in using the community driven ESLint builder, see: https://github.com/angular-eslint/angular-eslint#migrating-an-angular-cli-project-from-codelyzer-and-tslint.
[lint:html    ] 
[lint:html    ] [htmllint] found 0 errors out of 163 files
[lint:html    ] All files pass linting
[lint:prettier] src/app/shared/datatable/table/table.component.scss
[lint:tslint  ] Linting "ceph-dashboard"...
[lint:tslint  ] All files pass linting.
```

I'll re-trigger make check in case, but if it fails again, you may want to run `npm fix` for this file. Or make any manual changes as needed.

Reply:
Passing now.